spec/features/low_income_spec.rb

```diff
@@ -72,4 +75,20 @@
     expect(page).to have_text('Your request for Low Income has been rejected.')
     expect(page.html).to include(User.last.membership_code.code)
   end
+
+  scenario 'prerelease event and user has 1 available tickets but the low income window has closed' do
+    stub_eventbrite_event(available_tickets_for_code: 1, tickets_sold_for_code: 0)
+    create(:event, :prerelease, low_income_requests_end: Time.zone.now.advance(weeks: -1))
+    login
+
+    expect(page).to_not have_text('Apply for low income')
+  end
+
+  scenario 'live event and user has 1 available tickets but the low income window has closed' do
+    stub_eventbrite_event(available_tickets_for_code: 1, tickets_sold_for_code: 0)
+    create(:event, low_income_requests_end: Time.zone.now.advance(weeks: -1))
+    login
+
+    expect(page).to_not have_text('Apply for low income')
+  end
```

Comment:
I may have missed it, but is there a spec for if the event is no pre-release?
There's a lot of different logic paths in `low_income_open?` and I'd like to ensure they're all captured in the specs

Reply:
Also for when low income window hasn't opened yet? Are there others?

Reply:
@james what do you mena by no pre-release ? If there is no pre-release, there is no low income ticket link showing.

I did window closed, but not hadn't opened yet, just adding some tests.

Reply:
Sorry, I meant I can't see  spec for when `event_mode == 'prerelease'` if false, and therefore hides the low income ticket link.

Thanks.